utils/helper.sh

```diff
@@ -161,9 +161,9 @@ function check_running_cp() {
   check_curl
   expected_version=$1
 
-  actual_version=$( confluent local version | tail -1 | awk -F':' '{print $2;}' | awk '$1 > 0 { print substr($1,1,3)}' )
+  actual_version=$( confluent local version 2>/dev/null | awk -F':' '{print $2;}' | awk '$1 > 0 { print substr($1,1,3)}' )
```

Comment:
If `confluent local version` fails, this would cause it fail silently.  `confluent` was pulled out of CP 5.3, it's coming back in 5.4, but it's something to take into consideration.

Reply:
when you say `confluent local version` fails, what scenarios are you contemplating specifically (so I can test this change)?

Reply:
`confluent` bin isn't found?

Reply:
I added a new function to make this scenario very clear, see latest please.

I tested thus:

```
sh -c "source utils/helper.sh;PATH=/usr/bin;check_running_cp 5.4"

confluent command not found.  Please check your Confluent Platform installation
```